pkgs/development/python-modules/simplekml/default.nix

```diff
@@ -0,0 +1,19 @@
+{ lib , buildPythonPackage , fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "simplekml";
+  version = "1.3.1";
+  name = "simplekml-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.